gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-mqtt5/src/main/resources/schema/schema-form.json

```diff
@@ -23,6 +18,14 @@
             "title": "topic",
             "description": "Refers to an UTF-8 string that the broker uses to filter messages for each connected client. The topic consists of one or more topic levels. Each topic level is separated by a forward slash (topic level separator)."
         },
+        "reconnectAttempt": {
+            "type": "integer",
+            "title": "reconnectAttempt",
+            "description": "Number of reconnect attempt after any kind of disconnection.",
```

Comment:
```suggestion
            "description": "Number of reconnect attempts after any kind of disconnection.",
```

Reply:
May I change the property name also to its plurial?

Reply:
sure